scrapy/downloadermiddlewares/retry.py

```diff
@@ -70,8 +72,10 @@ def _retry(self, request, reason, spider):
             retryreq.meta['retry_times'] = retries
             retryreq.dont_filter = True
             retryreq.priority = request.priority + self.priority_adjust
+            self.stats.inc_value('retry/count')
```

Comment:
You may spot a retried status code that you didn't intend to (wrong setting perhaps), or a missing retried exception etc.